tests/test_memory_core/build_tb.py

```diff
@@ -1830,7 +1830,7 @@ def software_gold(app_name, matrix_tmp_dir, give_tensor=False, print_inputs=None
                            dump=matrix_tmp_dir, suffix=suffix, clean=clean, tensor_ordering=tensor_orderings['B'],
                            sparsity=sparsities_[0])
         c_mat = get_tensor(input_name='c', shapes=[shapes_[1]], give_tensor=give_tensor, tmp_dir=matrix_tmp_dir,
-                           dump=matrix_tmp_dir, suffix=suffix, clean=False, tensor_ordering=tensor_orderings['C'],
+                           dump=matrix_tmp_dir, suffix=suffix, clean=False, tensor_ordering=tensor_orderings['c'],
```

Comment:
So..you changed a big "C" to a little "c" for `tensor_orderings[]` index/key? Is that right?

Reply:
Yes this was the bug in the weekly regression. Since its a vector it should be a little "c". I am talking to Max and others about the way we test sparse applications, as it is pretty brittle right now.

Reply:
I see. C for matrix and c for vector. No problem.